UIforETW/UIforETWDlg.cpp

```diff
@@ -1131,7 +1940,7 @@ void CUIforETWDlg::OnSize(UINT nType, int /*cx*/, int /*cy*/)
 		int yDelta = windowRect.Height() - lastHeight_;
 		lastHeight_ += yDelta;
 
-		UINT flags = SWP_NOMOVE | SWP_NOOWNERZORDER | SWP_NOZORDER | SWP_NOACTIVATE;
+		const UINT flags = (SWP_NOMOVE | SWP_NOOWNERZORDER | SWP_NOZORDER | SWP_NOACTIVATE);

```

Comment:
I think the relevant definition of *clear* here is not whether something has a benefit that's obvious to the person proposing the change, but whether that benefit is *unanimously seen*.

Thus, by your observation, this change is *ipso facto* not worth the churn. Sharing thoughts is always a good idea.